ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx

```diff
@@ -49,8 +55,20 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
     <Typography variant="h2">{dashboardName}</Typography>
   );
 
-  const onSave = () => {
-    setEditMode(false);
+  const onSaveButtonClick = () => {
```

Comment:
I was just playing around some more and I think I found a bug. So my dashboard locally had JSON that was formatted in VSCode, when I made changes in the UI and then saved the dashboard, the raw JSON was saved, but all remaining lines after the first are still there, see screenshot:
![image](https://user-images.githubusercontent.com/9369625/206515867-fc963d41-e5c8-448e-9231-2b784b4573d4.png)
 
 Here is the data too after I clicked save: 
[DemoDashboardAlt.json.zip](https://github.com/perses/perses/files/10187687/DemoDashboardAlt.json.zip)

Reply:
this is fixed with #856